tests/test_checkout.py

```diff
@@ -229,11 +177,8 @@ def test_index_view(cart, status_code, url, rf, monkeypatch):
     assert response.url == url
 
 
-def test_checkout_discount(request_cart, sale, product_in_stock):
-    variant = product_in_stock.variants.get()
-    request_cart.add(variant, 1)
-    checkout = Checkout(request_cart, AnonymousUser(), 'tracking_code')
-    assert checkout.get_total() == TaxedMoney(
+def test_checkout_discount(checkout_with_items, sale):
+    assert checkout_with_items.get_total() == TaxedMoney(
         net=Money(5, 'USD'), gross=Money(5, 'USD'))
```

Comment:
Can we somehow get those values as a variables? Or maybe at least comment why it is just a particular number `5`.